package.json

```diff
@@ -93,11 +93,13 @@
     "listr": "^0.13.0",
     "loader-utils": "^1.1.0",
     "lodash": "^4.17.5",
+    "lodash.template": "^4.4.0",
```

Comment:
We already have `lodash` installed, so there's no need install `lodash.template`
You could do (just a suggestion, there are tons of way)
```js
const template = require('lodash/template');
```